lib/events/pgevents/pgevents.go

```diff
@@ -77,6 +77,37 @@ const (
 	AzureADAuth AuthMode = "azure"
 )
 
+const (
+	schemaV1Table = `CREATE TABLE events (
+		event_time timestamptz NOT NULL,
+		event_id uuid NOT NULL,
+		event_type text NOT NULL,
+		session_id uuid NOT NULL,
+		event_data json NOT NULL,
+		creation_time timestamptz NOT NULL DEFAULT now(),
+		CONSTRAINT events_pkey PRIMARY KEY (event_time, event_id)
+	);`
+	schemaV1CreationTimeIndexPg   = `CREATE INDEX events_creation_time_idx ON events USING brin (creation_time);`
+	schemaV1CreationTimeIndexCRDB = `CREATE INDEX events_creation_time_idx ON events (creation_time);`
```

Comment:
Can we just drop the index and the periodic background deletion for the crdb case, and rely on row-level TTL instead?